pkg/manifests/manifests.go

```diff
@@ -49,7 +49,7 @@ func (f *Factory) DefaultClusterIngress(ic *util.InstallConfig) (*ingressv1alpha
 	if err != nil {
 		return nil, err
 	}
-	ingressDomain := fmt.Sprintf("%s.%s", ic.Metadata.Name, ic.BaseDomain)
+	ingressDomain := fmt.Sprintf("*.apps.%s.%s", ic.Metadata.Name, ic.BaseDomain)
```

Comment:
This will break `TestDefaultClusterIngress`.  Also, do we want to publish the "*." in the canonical name in routes?

Reply:
No, canonical name is the name of the DNS wildcard root domain and API consumers should assume that it is `("%s.%s", myName, canonicalHostname)`.  I.e. canonical should always be reported as `a.b.c` and consumers must assume their name is `d.a.b.c`

Reply:
So we should add the "*." in `ensureDNSForLoadBalancer`.  Not sure about the "apps." though—probably should add "apps." here and adjust `TestDefaultClusterIngress` to keep it passing.

Reply:
We don't yet have any cluster config for the app domain, so I'm just making things up here. Do agree that the `*.` should be prefixed at runtime in `ensureDNSForLoadBalancer()`.